code/modules/clothing/factions/srm.dm

```diff
@@ -0,0 +1,154 @@
+	name = "saint-roumain machinist leather vest"
+	desc = "A modified Roumain leather duster with its large flaps and sleeves cut off to provide extra mobility when maintaining weapons and vessels belonging to the Church of Saint Roumain. Its specialty treatment grants it better protection against acid and fire."
+	icon = 'icons/obj/clothing/faction/srm/suits.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/suits.dmi'
+	icon_state = "armor_rouma_machinist"
+	item_state = "rouma_coat"
+	armor = list("melee" = 30, "bullet" = 20, "laser" = 20, "energy" = 40, "bomb" = 35, "bio" = 0, "rad" = 0, "fire" = 60, "acid" = 60)
+
+/obj/item/clothing/suit/armor/roumain/flamebearer
+	name = "saint-roumain flambearer robes"
+	desc = "A set of ashy-grey robes made from hard leather, adorned with gold trims. Its rough finish after a near-char and application of aromatics is heavily favored for the ecclesiastical sect of the Church of Saint Roumain, a living reminder of the Ashen Huntsman himself."
+	icon_state = "armor_rouma_flamebearer"
+
+/obj/item/clothing/suit/armor/roumain/colligne
+	name = "saint-roumain colligne coat"
+	desc = "A well-maintained hard leather coat typically worn to denote the rank of Colligne, a trainee Hunter Montagne. It is treated with bullet-resistant materials, and lined with the dark fur of Illestrenin dire wolves."
+	icon_state = "armor_rouma_colligne"
+	item_state = "rouma_coat"
+	body_parts_covered = CHEST|GROIN|ARMS|LEGS
+	cold_protection = CHEST|GROIN|LEGS|ARMS
+	heat_protection = CHEST|GROIN|LEGS|ARMS
+
+/obj/item/clothing/suit/armor/roumain/montagne
+	name = "saint-roumain montagne coat"
+	desc = "A stylish red coat to indicate that you are, in fact, a Hunter Montagne. Made of extra hard exotic leather, treated with bullet-resistant materials, and lined with the fur of some unidentifiable creature."
+	icon_state = "armor_rouma_montagne"
+	item_state = "rouma_montagne_coat"
+	body_parts_covered = CHEST|GROIN|ARMS|LEGS
+	armor = list("melee" = 30, "bullet" = 30, "laser" = 30, "energy" = 40, "bomb" = 25, "bio" = 0, "rad" = 0, "fire" = 70, "acid" = 90)
+	cold_protection = CHEST|GROIN|LEGS|ARMS
+	heat_protection = CHEST|GROIN|LEGS|ARMS
+
+///////////////
+//Spacesuits//
+//////////////
+
+/obj/item/clothing/head/helmet/space/hardsuit/solgov/roumain
+	name = "\improper roumain hardsuit helmet"
+	desc = "An armor-plated knight's helmet, the pointed design of it reminiscent of iconography depicting the great Ashen Huntsman."
+	icon = 'icons/obj/clothing/faction/srm/head.dmi'
+	mob_overlay_icon = 'icons/mob/clothing/faction/srm/head.dmi'
+	icon_state = "hardsuit0-roumain"
+	item_state = "hardsuit0-roumain"
+	hardsuit_type = "roumain"
+	worn_y_offset = 4
+
+/obj/item/clothing/suit/space/hardsuit/solgov/roumain
+	name = "\improper roumain hardsuit"
+	desc = "A modified knight's plate armor turned into a space-worthy hardsuit. A powered exoskeleton hidden behind the plates help grant it slight maneuverability."
```

Comment:
Suggested rewrite to drive home the artisanal nature of most SRM gear and make Hunter's Pride more than a little more explicitly badass: 
"A hand-crafted suit of armor either modified from a set of normal plate armor or designed to resemble one. A powered exoskeleton has been cleverly integrated into the design and, surprisingly, it is completely vacuum-proof. Suits like this are a testament to what the master craftsmen of Hunter's Pride are capable of."